CHANGELOG.md

```diff
@@ -5,11 +5,11 @@ All notable changes to the "azure-cosmosdb" extension will be documented in this
 ## 0.16.0 - 2021-02-23
 ### Added
 - Now depends on the "Azure Resources" extension, which provides a "Resource Groups" and "Help and Feedback" view
```

Comment:
@ejizba I just noticed this, is this supposed to be a continuation of this line?
""Report an Issue" button was removed from errors. Use the "Help and Feedback" view or command palette instead"

Reply:
I'm not entirely sure what you're asking, but I'll just say the way I did it was on purpose. If you give me a "suggested change", then I might be able to explain more

Reply:
I mean the starting of the sentence "Now depends..." didn't make sense. I was thinking something in the lines of "This extension now depends..." But I guess it is implied.